src/router.js

```diff
@@ -44,7 +53,7 @@ function createLocation({pathname = '', chain = [], params = {}, redirectFrom, r
       resolver
     )
   };
-}
+}e
```

Comment:
This `e` looks like a typo. Was it intentional?

Reply:
Fixed.